### PR TITLE
Stubs

### PR DESCRIPTION
**Pull request by sstanculeanu**

### CODE REVIEW

File: mock/pubSubSubscriptionStub.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package mock
```
Comment (SebastianMarian) [inline mock/pubSubSubscriptionStub.go:1]: Not needed for IsInterfaceNil ?

File: mock/pubSubTopicStub.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package mock
```
Comment (SebastianMarian) [inline mock/pubSubTopicStub.go:1]: Not needed for IsInterfaceNil ?

File: mock/pubSubSubscriptionStub.go (hunk 1 above)
Comment (sstanculeanu) [reply mock/pubSubSubscriptionStub.go:1]: not needed as the main implementation is the one from libp2p which does not have IsInterfaceNil

File: mock/pubSubTopicStub.go (hunk 2 above)
Comment (sstanculeanu) [reply mock/pubSubTopicStub.go:1]: this one too has the main implementation on libp2p, thus no IsInterfaceNil

**Review by bogdan-rosianu — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**